library/IvozProvider/Mapper/Sql/FaxesInOut.php

```diff
@@ -31,9 +31,8 @@ protected function _save(\IvozProvider\Model\Raw\FaxesInOut $model,
         $statusHaschanged = $model->hasChange("status");
 
         if ($isOutgoingFax && $statusHaschanged && $isPending) {
-            $faxSenderJob = new \IvozProvider\Gearmand\Jobs\FaxSender();
-            $faxSenderJob->setFaxInOut($model);
-            $faxSenderJob->send();
+            $ari = new \Asterisk\ARI\Connector();
+            $ari->sendFaxfileRequest($model);
```

Comment:
If request fails in all Application Servers, connector will throw an exception. It seems that exception is not required to be handled in the mapper, so I skipped the try..catch blocks in this section.

See 7e79e21